sdk/pinocchio/src/sysvars/tests/slot_hashes_tests.rs

```diff
@@ -0,0 +1,855 @@
+#[cfg(test)]
+use super::*;
+mod tests {
+    use super::*;
+    use core::mem::{align_of, size_of};
+    extern crate std;
+    #[allow(unused_imports)]
+    use std::vec::Vec;
```

Comment:
Should this import be conditional on the `std` feature?

Reply:
For convenience nostd *tests* can use vec to set up

Reply:
Vec is used in data setup functions (for clarity, convenience, and correctness) even when nostd tests are being run.